### ISSUE TITLE
Landing page copy seems too ambiguous

### ISSUE THREAD

**Issue by ianrandmckenzie** [QuiltMC/quiltmc.org#167]
Hi there,

I stumbled on Quilt after clicking around the Litematic listing on CurseForge and noticed your mod loader in its dropdown. I went to the site to find out what this is all about and reached the bottom of the landing page feeling like I didn't know much more than I started which was "It's most likely a mod loader and maybe has something to do with Fabric?"

I've done landing page copy for many of my clients now and am happy to help resolve this if we could turn this thread into a Q&A for me to get more info?

Thanks,

Ian

**Comment by Southpaw1496** [comment]
I'd recommend reading the [FAQ](https://quiltmc.org/en/about/faq) and [Newcommers Guide](https://quiltmc.org/en/about/newcomer-guide). Both of them could probably be a bit better highlighted, really.

**Comment by ianrandmckenzie** [comment]
That certainly seems to be a good start to tell potential users landing on the page. May I ask a few questions? After which I can submit a pull request for your review.

**Comment by ix0rai** [comment]
ask away!

**Comment by Southpaw1496** [comment]
If you're planning to submit a pull request it may be an idea to submit a draft request so that we can see a generated preview of your changes as you work.

What do you mean when you say "landing page copy"? If you mean a page based on a generic template, I don't think we'd be interested in that.

**Comment by ianrandmckenzie** [comment]
Re: Draft request, this would be my preferred option as well! Thanks for suggesting.

Re: Landing page copy, I mean landing page copy*writing* (speaking of ambiguity, sorry about that). While most of my clients are for-profit businesses whose profitability can often depend on a website landing page clearly and concisely, I still find it particularly useful as a user of other websites that the same thing be done.

So, before I list a deluge of personal biases that may have led to basically only me not understanding what Quilt is all about, perhaps I take a more agnostic approach and see if there's a consensus about whether or not the landing page accomplishes my opinion about what a landing page should have (which of course also requires an agreement about whether or not my opinions are correct/necessary for a website like this). So, based on my experience making and re-designing/writing landing pages that produce better results for the client and their customers, this is my opinion of what a landing page should have in question form:

* Note that ✅ (yes), 🤔 (maybe?), and ❌ (no) emojis are based on my opinion. I will leave my opinion out of "Bonus items"

### Above the Fold (The part of the page visible when landing on the page before scrolling)

- Does it clearly tell the visitor who you are? ✅
- Does it clearly tell the visitor what you do? ❌
- Bonus: Does it visually communicate who you are and what you do?

### Below the Fold

- **Is it clear what you want the visitor to do once on the landing page?** ❌ (There are way too many calls to action, I personally recommend two of them based on the 2 primary things that you need and what people want to do, my guess would be try to start testing Quilt beta as a user, and start mod development to grow the Quilt mod community. Keeping in mind I recognize that this may require a lot of thinking/reworking in the information architecture side of things)
- Is the page design and writing aligned what the visitor wants and what you as an organization want from the visitor? 🤔
- Does the page itself get people excited about installing the beta / developing a Quilt mod / whatever you want people to do once the land on this page?

I think the basis of Quilt is truly important but I think when it comes to people who are actually going to be users of Quilt, I'm not sure they care a whole lot. They just want to play modded Minecraft. That said, I'm sure developers would find the philosophy quite attractive. I've always avoided developing mods but after reading your FAQ I am 100% on board to start making the mods I've always wanted to make.

I won't do a pull request for now because I think there's probably a more substantial conversation about communication strategy that I have obviously not a part of. I do think I can help make things better but I also don't want to be negligent about past considerations since it is clear to me you all have put a ton of excellent work in your communication thus far.

**Comment by ianrandmckenzie** [comment]
Apologies if the above came as overly critical. I like what you all are doing here and would like to help clarify the messaging since Minecrafters come from all walks of life and may not know what a lot of words I see on the page. I guess I was inspired by the words on other pages and agree that being community driven and ease of use focused is crucial. 

When I go to Fabric or Forge as a young child or a tech-ignorant adult, I am either closing the site because I heard about it but can’t figure out what it is, or I have someone walking me through the steps telling me which buttons to click since I would be completely lost otherwise. If I’m lucky maybe a I found a YouTube video to help me do said clicking without a friend in my sidecar. 

My biggest problem with all of these mod loaders is how mysterious they are. I either have to use 3rd party organizations to abstract mod loaders into mod packs where I may not get the exact mods I want, or I am hail-marying a bunch of step by steps and not really knowing if it’ll work along the way. 

I think this paradigm shift you’re approaching could be taken further by making this process easier for the average Minecrafter by using simple language to say exactly what Quilt is and what they should do next whether they are a Minecraft player, a modder, or possibly a Minecraft player running into a bug (which can hopefully be designed to expedite them to contact the mod developer rather than the Quilt community since from what I’ve seen things are usually a matter of conflicts between two mods and not the environment itself)

I may be slow to respond right now, my home is undergoing repairs from a lot of water damage and even just loading GitHub on this connection took about 10mins to get to this page

**Comment by Southpaw1496** [comment]
Sorry for the late response, we work on this project on a volunteer basis and sometimes the obligations of life can get in the way.

> Apologies if the above came as overly critical
Not at all, I think it was well-articulated.

> I like what you all are doing here and would like to help clarify the messaging since Minecrafters come from all walks of life and may not know what a lot of words I see on the page

I agree that messaging can be confusing: I think we have an issue currently of assuming that anyone coming to Quilt will be familiar with Minecraft modding in general, which likely stems from our start as a fork of an existing loader with not much to recommend it other than the change of management. We now have several material advantages over Fabric, though most of them are developer-oriented rather than user-facing (new APIs etc.). Optimising for the "never modded Minecraft" before crowd could be good for us, as they won't have any baggage in the form of existing mods or workflows that aren't compatible with Quilt. However, it will be a challenge to make the website more accessible to them without coming across as condescending.

A lot of our development work happens on our [Discord Server](https://discord.quiltmc.org). If you would like to join, I can create a thread in the #community-tooling channel for us to discuss your ideas for a redesign of the homepage. I'm also happy to continue the discussion on GitHub if you would prefer.
